src/components/Tutorial/CodePreview.vue

```diff
@@ -158,13 +158,13 @@ export default {
       return this.hasRuntimePreview && this.isRuntimePreviewVisible;
     },
     runtimePreviewTitle() {
-      return this.hasRuntimePreview ? null : 'No preview available for this step.';
+      return this.hasRuntimePreview ? null : this.$t('tutorials.preview.no-preview-available-step');
     },
     togglePreviewText() {
-      return this.hasRuntimePreview ? 'Preview' : 'No Preview';
+      return this.hasRuntimePreview ? this.$tc('tutorials.preview.title', 1) : this.$tc('tutorials.preview.title', 0);
```

Comment:
(Feel free to ignore this if you don't think it's an improvement)

```suggestion
      return this.$tc('tutorials.preview.title', this.hasRuntimePreview ? 1 : 0);
```

Reply:
good improvement!